### PR TITLE
feat: user pvc to store the cached helm repo charts

### PR DESCRIPTION
**Pull request by 0xff-dev**
Fix: https://github.com/kubebb/core/issues/76
```shell
/opt/helm $ pwd
/opt/helm
/opt/helm $ ls
caches  config
/opt/helm $ ls caches/helm/repository/
kubebb-system.kubebb-charts.txt  kubebb-system.kubebb-index.yaml
/opt/helm $ ls config/helm/
repositories.lock  repositories.yaml
```

### CODE REVIEW

**Comment by bjwswang** [comment]
bump to a new version @0xff-dev

File: charts/kubebb-core/templates/deploy.yaml [hunk 1]
```diff
@@ -24,12 +24,26 @@ spec:
...
         - /manager
         image: {{ .Values.deployment.image }}
         imagePullPolicy: {{ .Values.deployment.imagePullPolicy }}
+        env:
+        - name: HELM_CACHE_HOME
+          value: /opt/helm/caches/helm
+        - name: HELM_CONFIG_HOME
+          value: /opt/helm/config/helm
+        - name: HELM_DATA_HOME
+          value: /opt/helm/helm
```
Comment (Abirdcfly) [inline charts/kubebb-core/templates/deploy.yaml:42]: Using the same naming scheme as above, here it should be `/opt/helm/data` ?

File: charts/kubebb-core/templates/pvc.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  storageClassName: standard
```
Comment (Abirdcfly) [inline charts/kubebb-core/templates/pvc.yaml:7]: maybe storageClass should be custom?

File: charts/kubebb-core/templates/pvc.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 5Gi
```
Comment (Abirdcfly) [inline charts/kubebb-core/templates/pvc.yaml:12]: maybe storage size should be custom?

**Review by Abirdcfly — APPROVED** [review]

**Review by bjwswang — APPROVED** [review]

**Pull request merged**